Frontend/Report/frontend-report-first-half-of-november-2024.md

```diff
@@ -0,0 +1,99 @@
+
+### Quick links
+
+- [Seamless React upgrades in micro-frontends](https://alexocallaghan.com/upgrading-react-with-microfrontends)
+- [React Navigation 7: Enhanced developer experience and powerful new features](https://reactnavigation.org/blog/2024/11/06/react-navigation-7.0/)
+
+## Next.js
+
+### [Exploring Next.js 15: What's new and why it's better than ever](https://aviral-blogs.hashnode.dev/exploring-nextjs-15-whats-new-and-why-its-better-than-ever)
+
+Next.js 15 enhances build speed, improves developer experience, and adds new features like full support for React Server Components and expanded App Router functionality. It also improves image optimization with dynamic resizing and global caching. 
+
+### [Advanced search param filtering](https://aurorascharff.no/posts/managing-advanced-search-param-filtering-next-app-router/)
+
+Comprehensive guide on implementing search parameter filtering using App Router with startTransition and useOptimistic.
+
+### Quick links
+
+- [Vercel simplifies Next.js self-hosting](https://thenewstack.io/vercel-makes-changes-to-next-js-to-simplify-self-hosting/)
+- [Integrating WordPress as a headless CMS with React or Next.js](https://scientyficworld.org/how-to-deploy-headless-wordpress-with-next-js/)
+
+## Others
+
+### [Why frameworks outperform Web components: A deep dive into key drawbacks](https://www.youtube.com/watch?v=UrS61kn4gKI)
+
+Explore the fundamental limitations of Web Components compared to modern frameworks like React, including issues with state management, integration challenges, server-side rendering, and real-world adoption. Understand why frameworks remain the practical choice for developers today.
+
+### [Enhancing web design with Container queries and the Containment API](https://www.joshwcomeau.com/css/container-queries-introduction/)
+Container queries enable responsive design by allowing elements to adjust based on their parent’s size. With widespread browser support, these tools promise to transform design practices, and the author will share practical examples in a future post.
+
+### Quick links
+
+- [Browser rendering process](https://abhisaha.com/blog/exploring-browser-rendering-process)
+- [HTML login form best practices](https://evilmartians.com/chronicles/html-best-practices-for-login-and-signup-forms)
+
+## Trending
+
+Growing trend in UI library development, focusing on separation of logic and presentation.
+
+### [TanStack Start: A Full-Stack React framework for modern web apps](https://tanstack.com/start/latest)
+
+TanStack Start is a powerful full-stack React framework featuring type-safe APIs, robust routing, full-document SSR, streaming, and server functions—perfect for building modern, scalable web applications.
+
+### [WebSocket optimization](https://www.recall.ai/post/how-websockets-cost-us-1m-on-our-aws-bill)
+
+Case study on optimizing WebSocket implementations for cost efficiency.
```

Comment:
Can we have more meaningful details here? Both the title & the description are too generic